mobile/src/bundles/talent/screens/bsa-badges/bsa-badges.tsx

```diff
@@ -46,6 +43,16 @@ const BsaBadges: React.FC = () => {
         return null;
     }, []);
 
+    const [checkedBadges, setCheckedBadges] = useState(
+        DEFAULT_VALUE_IS_CHECKED,
+    );
+    const handleToggleBadge = (badge: ValueOf<typeof BadgeType>): void => {
+        setCheckedBadges((previousChecked) => ({
+            ...previousChecked,
+            [badge]: !previousChecked[badge],
+        }));
+    };
+
```

Comment:
I changed something here because the Checkbox no longer contains control and name.
This page will be rewritten in another task.